helper/src/components/previewDialog.js

```diff
@@ -1,34 +1,36 @@
 import React from "react";
 import { useBoolean } from "@fluentui/react-hooks";
-import { Dialog, DialogType, DialogFooter } from '@fluentui/react/lib/Dialog';
-import { PrimaryButton,DefaultButton } from '@fluentui/react/lib/Button';
+import { Dialog, DialogType, DialogFooter } from "@fluentui/react/lib/Dialog";
+import { PrimaryButton, DefaultButton } from "@fluentui/react/lib/Button";
 
-export function PreviewDialog({previewLink}) {
-
-  const [hideDialog, { toggle: toggleHideDialog }] = useBoolean(false);
+export function PreviewDialog({ previewLink, onClose, show}) {
+  // const [hideDialog, { toggle: toggleHideDialog }] = useBoolean(false);
 
   const dialogContentProps = {
     type: DialogType.normal,
-    title: 'Preview Feature',
-    closeButtonAriaLabel: 'Close',
+    title: "Preview Feature",
+    closeButtonAriaLabel: "Close",
     subText: `Review the instructions on this page ${previewLink} to enable the feature `,
```

Comment:
previewLink is a url, we should be displaying it as a clickable <a> element

Reply:
it seems the template literals don't evaluate html if i try to make this clickable